.gitignore

```diff
@@ -3,3 +3,4 @@ build/*.js
 npm-debug.log
 .DS_Store
 package-lock.json
+.env
```

Comment:
does your local `.env` file include the `ACCESS_SECRET_TOKEN`? 
Makes sense not to include it GitHub, but we'll need to share that with the team so we can run it locally